tools/cjxl_main.cc

```diff
@@ -584,7 +584,7 @@ jxl::Status GetPixeldata(const std::vector<uint8_t>& image_data,
 #endif
 #if JPEGXL_ENABLE_JPEG
     if (jxl::extras::DecodeImageJPG(encoded, color_hints, size_constraints,
-                                    &ppf)) {
+                                    /*output_bit_depth=*/8, &ppf)) {
```

Comment:
Why not 16 here?

Reply:
Right, it can be 16 now. I just forgot that I made it work with the old library in the meantime.